src/main.cc

```diff
@@ -710,6 +711,13 @@ main(int argc, char **argv)
 	memtx_tx_manager_init();
 	crypto_init();
 	systemd_init();
+
+	const int override_cert_paths_env_vars = 0;
+	int res = ssl_cert_paths_discover(override_cert_paths_env_vars);
+	if (res != 0)
+		panic_syserror("No enough memory for setup "
```

Comment:
Are you sure panic in this case is a good idea? I believe warning will be enough.

Reply:
Fixed (used `say_warn` instead of `panic_syserror`)